### PR TITLE
Updating protos to latest, adjusting examples where necessary

### PR DESCRIPTION
**Pull request by mkr**
* Latest protos added import of `protoc-gen-openapiv2/options/annotations.proto`.
* Adjusted examples to incorporate this change.

### CODE REVIEW

File: language-examples/csharp/grpc/README.md [hunk 1]
```diff
@@ -23,16 +23,18 @@ Please see the details here: [Authentication](../../../README.md).
 Following are the steps that need to be done to run this example:
 
 1. Clone the repo (Please see details here: [cloning guidelines](../../../README.md)).
-2. cd to `vectara-demo/csharp/grpc` directory.
-3. Run the `download_google_protos.sh` script on terminal like following:
+2. cd to `vectara-demo/csharp/common` directory and run `dotnet build common.csproj`. This will build the authentication
```
Comment (aamirbutt) [inline language-examples/csharp/grpc/README.md:26]: Can you please change the `vectara-demo` to `language-examples`

File: language-examples/php/grpc/README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ To run the example, do the following:
 
     `sh generate_proto_stubs.sh`
    
-   This will generate two folders containing Vectara proto php stubs: `Com/Vectara/` and `GPBMetadata/`
+   This will generate threee folders containing Vectara proto php stubs: `Com/Vectara/`, `GPBMetadata/`, and `Grpc/`
```
Comment (aamirbutt) [inline language-examples/php/grpc/README.md:47]: Typo in `three`

**Review by aamirbutt — COMMENTED** [review]
Left two minor comments. Can you please address those?
Thanks for the PR.

File: language-examples/csharp/grpc/README.md (hunk 1 above)
Comment (mkr) [reply language-examples/csharp/grpc/README.md]: Addressed in 4dee05582ebc92ad5876afa58d2409b04fcd5252

File: language-examples/php/grpc/README.md (hunk 2 above)
Comment (mkr) [reply language-examples/php/grpc/README.md]: Addressed in 4dee05582ebc92ad5876afa58d2409b04fcd5252

**Comment by mkr** [comment]
Thank you @aamirbutt. I have rebased to main and addressed your comments.

**Pull request merged**